app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
          :confirmable
 
   validates :email , email: {mode: :strict}
+  validates :username , length: { minimum: 4 , maximum: 16 }
```

Comment:
Šeit vari viena rindā:
  validates :username, presence: true, length: { minimum: 4 , maximum: 16 }

Rails to prot!

Reply:
For some reason nebija iesavojies ,  jo im sure ka biju ta izdarijis , bet ok i guess ill do it again 😄